modules/user_accounts/php/edit_user.class.inc

```diff
@@ -416,58 +417,6 @@ class NDB_Form_User_Accounts extends NDB_Form
             //END EXAMINER UPDATE
         }
 
-        // START NOTIFICATIONS UPDATE
-        if ($this->page==='my_preferences') {
-            // get current notifications for user
-            $curr_sub = NDB_Notifier::getUserNotificationModuleServices(
-                $user->getData("ID")
-            );
-
-            if (!$this->isCreatingNewUser()) {
-                // get notification details
-                foreach ($curr_sub as $module => $operations) {
-                    foreach ($operations as $operation => $services) {
-                        $module_id = NDB_Notifier::getNotificationModuleID(
-                            $module,
-                            $operation
-                        );
-                        unset($services['desc']);
-                        foreach ($services as $service => $subscribed) {
-                            $service_id = NDB_Notifier::getNotificationServiceID(
-                                $service
-                            );
-                            $var_name   = "notif_" . $module . "_"
-                                . $operation . "_" . $service;
-
-                            if ($values[$var_name] === 'on') {
-                                if ($subscribed === 'N') {
-                                    $DB->insert(
-                                        'users_notifications_rel',
-                                        array(
-                                         "module_id"  => $module_id,
-                                         "service_id" => $service_id,
-                                         "user_id"    => $user->getData("ID"),
-                                        )
-                                    );
-                                }
-                            } else {
-                                if ($subscribed === 'Y') {
-                                    $DB->delete(
-                                        'users_notifications_rel',
-                                        array(
-                                         "module_id"  => $module_id,
-                                         "service_id" => $service_id,
-                                         "user_id"    => $user->getData("ID"),
-                                        )
-                                    );
-                                }
-                            }
-                            unset($values[$var_name]);
-                        }
-                    }
-                }
-            }
-        }
```

Comment:
can't comment on it, but I think the line below here is meant to be removed (you removed the "START NOTIFICATIONS" comment at the start..)